test/language/expressions/equals/bigint-and-incomparable-primitive.js

```diff
@@ -0,0 +1,26 @@
+// Copyright (C) 2017 Josh Wolfe. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+description: Non-strict equality comparison of BigInt and miscellaneous primitive values
+esid: sec-equality-operators-runtime-semantics-evaluation
+info: |
+  EqualityExpression : EqualityExpression == RelationalExpression
+    ...
+    5. Return the result of performing Abstract Equality Comparison rval == lval.
```

Comment:
This spec text doesn't seem very helpful to me. Is there something better to say here?

Reply:
Unfortunately, because you are testing false cases, I don't think there is more to say in the info. 

Thought experiment: 
(1) You could go an extreme route and copy the whole Abstract Equality Comparison to see that you are testing negative cases (cases that are not listed explicitly in the algorithm), but that potentially noisy/distracting. A curious tester could just look up Abstract Equaliity Comparison in the test themselves. 
(2) you could just include the "return false" line you had before, which is also a bit confusing (if the specs change -- specifically, if the line number changes and there is more than one "return false")

So, it seems like there is not much more to add to the info. Nonetheless your question made me realize the description of the "info" tag in the CONTRIBUTING.md file is out of date with the current best practices, so I'll fix that later!

Also this is just my opinion, and I only just started working on test262, other reviewers might have a different idea of what to include the info :)

Reply:
(1), yes , the whole thing is certainly too much and distracting. 
(2), exactly, also the initial operation might be expanded or changed the Return false instruction might become even more unclear.

Thanks for volunteering on to improve the contributing, my bias with the project makes me assume the documentation is fine, fresh eyes are always great for improvements.